Cargo.toml

```diff
@@ -12,10 +12,11 @@ readme = "README.md"
 autobenches = false
 
 [features]
-default = ["elf", "pe"]
+default = ["elf", "pe", "cmdline"]
 bzimage = []
 elf = []
 pe = []
+cmdline = []
```

Comment:
We should make it possible to build without any features. Adding `cmdline` as a feature is not necessary, and increases the complexity.